tests/foreman/cli/test_errata.py

```diff
@@ -225,6 +239,7 @@ def errata_hosts(register_hosts):
             result = host.execute(f'yum install -y {old_package}')
             if result.status != 0:
                 pytest.fail(f'Failed to install {old_package}: {result.stdout} {result.stderr}')
+        host.execute('subscription-manager repos')
```

Comment:
What's the point of running this line? If it's for debugging / logging I would say use `logger.info()`

Reply:
It shouldn't be necessary here since it is correctly called earlier before removing and applying errata- [L735](https://github.com/SatelliteQE/robottelo/blob/3f98a1578cb6a0ec71e9f772653a7b7f29bae4c0/tests/foreman/cli/test_errata.py#L735). 
My understanding is this is called after host registration to update subscription-manager package status and errata applicability.

Reply:
Ok makes sense thanks!